pkg/events/events.go

```diff
@@ -5906,10 +5905,7 @@ var Definitions = eventDefinitions{
 					{EventID: SharedObjectLoaded},
 					{EventID: SchedProcessExec}, // Used to get mount namespace cache
 				},
-				Capabilities: []cap.Value{
-					cap.SYS_PTRACE,   // Used to get host mount NS for bucket cache
-					cap.DAC_OVERRIDE, // Used to open files across the system
-				},
+				Capabilities: []cap.Value{},
```

Comment:
Why are we removing required dependencies?
One of the reasons to have required dependencies is also to check if we have them during init time and to document which events have capability depndencies

Reply:
Capabilities removed from required dependencies were raised as "specific" in specific places (as the code isn't related to a specific event). Capabilities kept as required are the ones required to be present in the base ring only and related to a specific event.

Example: "Used to open files across the system" is very generic and does not translate the difference among the base ring concept (needed so the pipeline logic works without specific privilege raising) VERSUS the specific parts of tracee that need raising/dropping privileges (reading signature files, mounting cgroupfs, reading /proc/modules (FillModulesFromProcFS), all parts of the code unrelated to a specific event).

Reply:
I understand, but I think it will raise a problem when we will want to drop capabilties on init (including from bounding set) of the capabilities not used by Tracee since now we can't know which capabilities are required at init time. Isn't that so?

Reply:
Maybe we should add a new field to the capability dependecies declaration which will say if the capability is required to be part of the base ring or not.
This way, we will still be able to know which events actually require which capabilities, and also keep the base ring concept introduced in this PR

Reply:
commit 15b668ce (HEAD -> policies, rafaeldtinoco/policies)
Author: Rafael David Tinoco <rafaeldtinoco@gmail.com>
Date:   Mon Apr 10 16:36:57 2023

    capabilities: per ring type capabilities dependencies

And, if you note, CAP_NET_ADMIN now is part of eBPF ring only and won't be in the "base" ring, so your change is welcome and better I believe.